docs/static/ds6/skin.css

```diff
@@ -1190,13 +1190,13 @@ span.combobox__icon {
   border-color: transparent;
 }
 .page-notice {
+  -webkit-box-align: stretch;
+      -ms-flex-align: stretch;
+          align-items: stretch;
```

Comment:
Do we like to have these "lined up" like this? Does the linter not care?

Reply:
Ah, this is part of the dist. Never mind.

Reply:
Yeah, autoprefixer does this formatting. It might be configurable. I'll check.

Reply:
No worries. If it's generated I'm not too worried about it.